assets/i18n/fi.json

```diff
@@ -610,7 +610,6 @@
 	"I18N_LEARNER_GROUP_SYLLABUS_ITEM_NOT_STARTED_YET": "Ei vielä aloitettu",
 	"I18N_LEARNER_GROUP_SYLLABUS_LESSONS": "opoitunnit",
 	"I18N_LEARNER_GROUP_SYLLABUS_TAB": "Opetussuunnitelma",
-	"I18N_LEARNER_GROUP_VIEW_DETAILS": "Näytä tiedot",
 	"I18N_LEARNER_GROUP_VIEW_PREFERENCES": "RYHMÄN ASETUKSET",
```

Comment:
Hi, @heyimShivam  Thanks for review.
1) I18N_LEARNER_GROUP_VIEW_DETAILS is now changed to I18N_LEARNER_GROUP_MORE_INFO so I have removed the view-detail and instead of i used more info as per the mocks.
2)FYI, When we include the new I18N string , we only include in en.json and qqq.json and rest of the files were modified by others.